lib/main.py

```diff
@@ -77,9 +77,10 @@ def run(self, *args, **kwargs):  # pylint: disable=unused-argument
                 )
             except Exception as e:  # noqa
                 print(str(e), flush=True)
-                nc = NextcloudApp()
-                nc.log(LogLvl.ERROR, str(e))
-                nc.providers.task_processing.report_result(task["id"], error_message=str(e))
+                with suppress(Exception):
+                    nc = NextcloudApp()
+                    nc.log(LogLvl.ERROR, str(e))
+                    nc.providers.task_processing.report_result(task["id"], error_message=str(e))
```

Comment:
this should serve the purpose, no? Or did you mean with the logger module?

Reply:
I mean, sometimes it's useful to output exceptions to logs, at least to debug or "info" level - so that it would be easier to diagnose problems later.

I also like "supress", a very good thing.

Reply:
oh, yes logging there would be good. I was thinking along the lines that there was already an error that should be addressed, others were not relevant. Logging again doesn't hurt :v: